Mathlib/LinearAlgebra/Multilinear/Basic.lean

```diff
@@ -1734,42 +1733,21 @@ theorem curryFinFinset_symm_apply_piecewise_const {k l n : ℕ} {s : Finset (Fin
     rw [finSumEquivOfFinset_inr, Finset.piecewise_eq_of_not_mem]
     exact Finset.mem_compl.1 (Finset.orderEmbOfFin_mem _ _ _)
 
-@[simp]
-theorem curryFinFinset_symm_apply_piecewise_const_aux {k l n : ℕ} {s : Finset (Fin n)}
```

Comment:
These are unused, and only existed in the first place to satisfy a linter warning. Easier if we'd just removed `@[simp]` to begin with!

Reply:
Just flagging this removed lemma, in case you want to deprecate it.

Reply:
I think we're okay to not deprecate `_aux` lemmas.